modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -907,16 +907,16 @@ def mean(self, axis, **kwargs):
         # TODO-FIX: this function may work incorrectly with user-defined "numeric" values.
         # Since `count(numeric_only=True)` discards all unknown "numeric" types, we can get incorrect
         # divisor inside the reduce function.
-        def map_fn(df, **kwargs):
+        def map_fn(df, numeric_only=False, **kwargs):
```

Comment:
Why do we need this change?

Reply:
> Which part of the code passes this argument to `map_fn` then?

When calling `mean` function through `**kwargs`.